### PR TITLE
The Combat Code Cleanup Cleanup Part 1

### LINKED ISSUE #13174: Attacking as a mob with no attack damage produces a "miss" sound
@unid15 's combat dealio probably

So if you take a mouse or a cat for example and try to attack someone you get
```The mouse (67) has missed the unknown!```

since their damage lower and upper are both zero. Regular simple animals are unaffected

### PR DESCRIPTION
**Pull request by unid15**
Addresses most of issues in #13061

Will test and finish tomorrow

Fixes #13174

### CODE REVIEW

**Comment by Probe1** [comment]
Thank you.

File: code/modules/mob/living/carbon/human/combat.dm [hunk 1]
```diff
@@ -27,51 +27,42 @@
...
 	var/datum/organ/external/affecting = get_organ(ran_zone(zone_sel.selecting))
-	if(target.disarmed_by(src))
+	if(prob(40))
+		playsound(loc, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
+		visible_message("<span class='danger'>[src] has attempted to disarm [target]!</span>")
 		return
 
-	var/randn = rand(1, 100)
-	if(randn <= 25)
+	if(prob(40)) //40% to push
```
Comment (9600bauds) [inline code/modules/mob/living/carbon/human/combat.dm]: 40?

Comment (unid15) [reply code/modules/mob/living/carbon/human/combat.dm]: I'll have to clarify that in the comment

There's a 60% chance of not missing and a 40% chance of stunning (if you don't miss). 0.6 * 0.4 = 24% chance of stunning per click, roughly same as before

**Comment by ComicIronic** [comment]
You don't have to put defines in their own file unless they're shared between files. If a define is only used in one file, it can go at the top.

**Comment by 9600bauds** [comment]
This is shared between files, that's the problem

**Comment by ComicIronic** [comment]
Where's the second occurance?

**Comment by 9600bauds** [comment]
Oh whoops wrong PR

**Comment by unid15** [comment]
Now fixes #13174 and removes mice's ability to spam chat (mouse-sized mobs no longer create any sorts of messages when clicking on another mob)

**Pull request merged**